core/src/main/resources/defaults/blueprints/7.2.2/cdp-data-engineering-spark3.bp

```diff
@@ -148,15 +148,6 @@
             "refName": "yarn-GATEWAY-BASE",
             "roleType": "GATEWAY",
             "base": true,
-            "configs": [
```

Comment:
`configs` is removed as well

Reply:
Thanks Krisztian for catching this. I have fixed this.